engine/apps/slack/scenarios/alertgroup_appearance.py

```diff
@@ -1,28 +1,46 @@
 import json
+import typing
 
 from apps.api.permissions import RBACPermission
 from apps.slack.scenarios import scenario_step
+from apps.slack.types import (
+    Block,
+    BlockActionType,
+    EventPayload,
+    InteractiveMessageActionType,
+    ModalView,
+    PayloadType,
+    RoutingSteps,
+)
 
 from .step_mixins import AlertGroupActionsMixin
 
+if typing.TYPE_CHECKING:
+    from apps.slack.models import SlackTeamIdentity, SlackUserIdentity
+
 
 class OpenAlertAppearanceDialogStep(AlertGroupActionsMixin, scenario_step.ScenarioStep):
     REQUIRED_PERMISSIONS = [RBACPermission.Permissions.CHATOPS_WRITE]
 
-    def process_scenario(self, slack_user_identity, slack_team_identity, payload):
+    def process_scenario(
+        self,
+        slack_user_identity: "SlackUserIdentity",
+        slack_team_identity: "SlackTeamIdentity",
+        payload: EventPayload,
+    ) -> None:
         alert_group = self.get_alert_group(slack_team_identity, payload)
         if not self.is_authorized(alert_group):
             self.open_unauthorized_warning(payload)
             return
 
         private_metadata = {
-            "organization_id": self.organization.pk if self.organization else alert_group.organization.pk,
```

Comment:
while adding typing it became evident that this line can be simplified. AFAIK `AlertGroup` model does not have an `organization` attribute, `@property`, nor do any of its subclasses contain one